common/overlay/overlay.go

```diff
@@ -23,6 +23,10 @@ import (
 	"github.com/hashicorp/errwrap"
 )
 
+// sanitizer defines a string translator used to escape colon and coma
```

Comment:
Mino grammar issue here and in other place in this commit: "comma" (with double m).